styles/stylesheet.css

```diff
@@ -0,0 +1,422 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Arial, Helvetica, sans-serif;
+    box-sizing: border-box;
+    font-size:100%;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+label,
+legend {
+    color: rgb(20, 66, 109);
+    font-weight: bold;
+}
+
+h2 {
+    margin-top: 1rem;
+}
+
+h2,
+h3,
+h4,
+.dogs img,
+.dogs p,
+.all-dogs p {
+    margin-bottom: 1rem;
+}
+
+.blog h2,
+.blog p {
+    margin-top: 0;
+    margin-bottom: 1rem;
+}
+
+h3,
+h4,
+.dogs p,
+.all-dogs p,
+.dogs button
+    {
+    margin-left: 1rem;
+    margin-right: 2rem;
+}
+
+.dogs-adopt h4,
+.cost h4 {
+    margin-left: 0rem;
+    margin-top: 1rem;
+}
+
+
+header {
+    background-color: black;
+    color: white;
+    background-image: url("../images/header-image.jpg");
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.logo {
+    font-weight: bold;
+}
+
+.cart {
+
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: center;
+    font-size: .8rem;
+}
+
+header h1 {
+    color: #008A00;
+    font-style: italic;
+    font-size: 1.25rem;
+}
+
+.hdr-left {
+    margin-left: 5rem;
+    margin-top: 1rem;
+    margin-bottom: 3rem;
+}
+
+.hdr-right {
+    margin-right: 7rem;
+    margin-top: 3rem;
+    margin-bottom: 3rem;
+}
+
+header button {
+    padding: .5rem 2rem;
+    background-color: #008A00;
+    border: 1px solid rgb(48, 44, 44);
+    color: white;
+    font-weight: bold;
+    font-size: .7rem;
+}
+
+header p {
+    font-size: .7rem;
+    padding-top: 1rem;
+}
+
+header img,
+header p,
+header button {
+    margin-top: .5rem;
+}
+
+.cart img {
+    width: 30%;
+    margin-right: 1rem;
+    margin-left: 1rem;
+}
+
+nav {
+    background-color: lightgray;
+}
+
+nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-content: flex-start;
+    flex-wrap: wrap;
+    padding-left: 5rem;
+}
+
+nav li,
+footer li,
+nav p {
+    list-style: none;
+    padding: 1rem 0rem 1rem 1rem;
+    font-size: .75rem;
+    font-weight: bold;
+}
+
+nav a,
+footer a {
+    text-decoration: none;
+}
+
+
+.wrapper,
+.blogs {
+    margin-left: 5rem;
+    /* margin-right: 5%; */
+    /* background-color: lightblue; */
+}
+
+.intro {
+    display: grid;
```

Comment:
Right now, when I shrink your page down. Your paragraph text overlaps the image. I would suggest only using CSS grid for your dog grid. Then use flexbox for cases like this. You'll find it will simplify your code here and make this part of the site super responsive.